tests/utilities/test_all_gather_grad.py

```diff
@@ -54,7 +54,7 @@ def _test_all_gather_ddp(rank, world_size):
     assert torch.allclose(grad2, tensor2.grad)
 
 
-@RunIf(skip_windows=True, skip_49370=True)
+@RunIf(skip_windows=True, skip_49370=True, skip_hanging_spawn=True)
```

Comment:
why not add a min PyTorch and python requirements in the test with a comment instead of a separate flag?

Reply:
I feel like this is cleaner. Makes it easy to find the tests later

There's nothing guaranteeing that all the comments will be equal in the future if more tests are skipped. It also runs the risk of somebody forgetting one of the 2.

And we can add and remove code from tests no problem, so BC is not a concern.

Reply:
if we skip more tests with this flag, then we also cannot guarantee that they will fall under the same version constraints.

but these are tests, not for users.. so its fine I guess.